ocs_ci/ocs/resources/pod.py

```diff
@@ -357,40 +357,27 @@ def get_ceph_tools_pod():
     return ceph_pod
 
 
-def get_rbd_provisioner_pod():
+def get_csi_provisioner_pod(interface):
```

Comment:
Looks like this function name should be changed, as commented by Raz earlier.

Reply:
I don't understand. do you mean the function name should instead be `get_provisioner_pod` and not `get_csi_provisioner_pod`?